apps/src/templates/instructions/InlineAudio.jsx

```diff
@@ -159,27 +157,27 @@ const InlineAudio = React.createClass({
     const contentPath = `${hash}/${encodeURIComponent(message)}.mp3`;
 
     return `${TTS_URL}/${voicePath}/${contentPath}`;
-  },
+  }
 
-  toggleAudio: function () {
+  toggleAudio = () => {
```

Comment:
Why do the toggle methods need to be arrow functions?

Reply:
One of the changes when you switch from `createClass` to ES6 is that React no longer autobinds instance methods to the component instance, so if they get passed around as callbacks you have to bind them some other way.

`toggleAudio` is passed as a click callback here:

https://github.com/code-dot-org/code-dot-org/blob/c44242b3e038d6f31812f4440ade3f1b40b44863/apps/src/templates/instructions/InlineAudio.jsx#L201

Class property syntax and arrow functions is a nice way to autobind instance methods.